src/components/Workflow/Workflow.js

```diff
@@ -100,6 +100,11 @@ const Workflow = ({
     setElements(getLayoutedElements(nodes.concat(edges)))
   }, [selectedJob.uid, workflow])
 
+  const getCloseDetailsLink = location => {
+    const urlArray = location.pathname.split('/')
+    return urlArray.slice(0, urlArray.length - 2).join('/')
```

Comment:
`Array.prototype.slice` supports getting negative integers as values for `start` and `end` parameters, which means counting from the end of the array.
```suggestion
    return urlArray.slice(0, -2).join('/')
```